omise/omise.php

```diff
@@ -40,4 +40,100 @@ public function __construct()
     }
+
+    public function getContent()
+    {
+        $fields_form[0]['form'] = array(
+            'legend' => array(
+                'title' => $this->l('Settings')
+            ),
+            'input'  => array(
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Enable/Disable'),
+                    'name'     => 'module',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enable Omise Payment Module.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'module_enabled',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'module_disabled',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Sandbox'),
+                    'name'     => 'sandbox',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enabling sandbox means that all your transactions will be in TEST mode.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'sandbox_on',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'sandbox_off',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for test'),
+                    'name'     => 'publicKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Secret key for test'),
+                    'name'     => 'secretKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode secret key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for live'),
+                    'name'     => 'publicKeyForLive',
+                    'required' => false,
+                    'desc'     => 'The "Live" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
```

Comment:
@nimid got it,
also just checked from Prestashop code,
```html
<div class="input-group fixed-width-lg">
    <span class="input-group-addon">
        <i class="icon-key"></i>
    </span>
    <input type="password"
        id="{if isset($input.id)}{$input.id}{else}{$input.name}{/if}"
        name="{$input.name}"
        class="{if isset($input.class)}{$input.class}{/if}"
        value=""
        {if isset($input.autocomplete) && !$input.autocomplete}autocomplete="off"{/if}
        {if isset($input.required) && $input.required } required="required" {/if} />
</div>
```
it's impossible to retrieve data and assign to the `password` field's value with that array configuration, they left `value` attr as `value=""` 😶

can I jump back to my previous question, 
"But just curious, why we have to shift it to another PR? 
Cuz the PR topic is about implement setting page, and if you gonna change secret key you have to reimplement setting page again, so why we can't finish in this PR?"

(Btw, for me, implement with their role/permission system is good idea, but just afraid that it gonna take time for first release.)